### DISCUSSION TITLE
[2021.09.13] 이름이 없는 동물의 아이디

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#125] [SQL]
[이름이 없는 동물의 아이디](https://programmers.co.kr/learn/courses/30/lessons/59039)

**Comment by hyk339** [comment]
[이름이 없는 동물의 아이디 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59039.sql)

**Comment by leechang4** [comment]
[이름이 없는 동물의 아이디 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59039.sql)

**Comment by pyunji** [comment]
[이름이 없는 동물의 아이디 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59039.sql)

**Comment by chaeyeon4u** [comment]
[이름이 없는 동물의 아이디 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59039.sql)
